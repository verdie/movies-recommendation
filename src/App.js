import React from 'react';
import {Route} from 'react-router-dom'
import GenresList from './components/GenresList';
import GenresItem from './components/GenresItem'
import Recommendation from './components/Recommendation'

class App extends React.Component {
  render() {
    return (
      <main>
        <Route path='/' exact component={GenresList}/>
        <Route path='/genre/:genreId' exact component={GenresItem}/>
        <Route path='/recommendation' exact component={Recommendation}/>
      </main>
    )
  }
}

export default App;
