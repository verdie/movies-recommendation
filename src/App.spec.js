import React from 'react'
import { render } from 'enzyme'
import App from './App'
import {BrowserRouter} from 'react-router-dom'
import toJson from 'enzyme-to-json'

describe('App component', () => {
it('renders correctly', () => {
  const wrapper = render(<BrowserRouter><App /></BrowserRouter>);
  expect(toJson(wrapper)).toMatchSnapshot();
})
})
