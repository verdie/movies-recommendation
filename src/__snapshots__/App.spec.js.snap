// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders correctly 1`] = `
<main>
  <div
    class="wrap"
  >
    <h2>
      Please select category that you like
    </h2>
    <div
      class="category-wrapper"
    />
  </div>
</main>
`;
